### PR TITLE
Add an option to disable writing to .p4ticket during authentication.

### PR DESCRIPTION
**Pull request by earthling**
P4Java and the p4 command line tool both lock this file during updates. This can cause the scm plugin analysis to fail when analyses are conducted concurrently on the same machine (i.e., one or more may fail with permission denied errors). This pull request adds an option to disable writing to the tickets file.

### CODE REVIEW

**Comment by henryju** [comment]
We usually don't like adding configuration flags, so what about always skipping writing of ticket? Is there any reason to let the plugin write ticket?

**Comment by earthling** [comment]
I can't see a reason for the sonar-scm-perforce plugin to write this file. It's really just a side effect of authenticating with p4java. I think in most scenarios, the .p4tickets file will have been created ahead of time when the source code is checked out for analysis (by p4 or p4v). I'll remove the option.

**Comment by earthling** [comment]
For what it's worth, I've also been running this [pull request](https://github.com/SonarQubeCommunity/sonar-scm-perforce/pull/12) in our environment. Haven't seen any issues.

**Comment by henryju** [comment]
@earthling As you may know we moved this plugin under "SonarQubeCommunity" umbrella since we have not the (human) bandwidth at SonarSource to test Perforce. Your P/R look good to me but I will not test it. I could merge the P/R but I think it would be easier if you could take the lead on this plugin. Of course I will stay available for review if you want.
TL;DR would you accept to take the lead on this plugin?

**Comment by earthling** [comment]
Sure, I will take the lead on this plugin.

**Comment by earthling** [comment]
I don't have write permissions to this repo.

**Comment by henryju** [comment]
Sorry, I was also missing some permissions. You should now be able to commit. Feel free to update the pom.xml:
- declare yourself as main developer
- change organization

Once your changes are merged, if you want to release the plugin on SonarQube update center, please read: http://docs.sonarqube.org/display/DEV/Deploying+to+Update+Center
